Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -858,27 +903,27 @@
         isAutoPostBackEnabled: false,
         extraParams: {},
         // re-motion: changed selectFirst from boolean field to function
-        selectFirst: function(inputValue, searchTerm) { return true; },
-        formatItem: function(row) { return { html : row[0], class : null, isAnnotation : false }; },
+        selectFirst: function(inputValue: string, searchTerm: string) { return true; },
+        formatItem: function(row: JQuery) { return { html : row[0], class : null, isAnnotation : false }; },
```

Comment:
why is row a `JQuery` object instead of string.

@a-ctor I'm starting to wonder if we don't want to do a follow-up task where we replace a ton of those strings with value types to add some kind of typesafety.